### DISCUSSION TITLE
Is it possible set Top search bar collapsed by default?

### DISCUSSION THREAD

**Discussion by baur** [grafana/grafana#67650] [General]
Is it possible set Top search bar collapsed by default?

before

<img width="960" alt="image" src="https://user-images.githubusercontent.com/2898361/235659978-fd5ee765-9260-42ab-8fc4-c873659b9028.png">

after (collapsed by default)

<img width="960" alt="image" src="https://user-images.githubusercontent.com/2898361/235660063-2db4ab5b-e86a-4638-98bc-ec4ff048e45a.png">

or hide command palette at all

**Comment by ashharrison90** [comment]
@baur either:
- via the UI, click the monitor button next to the arrow when collapsed
- via url params, use `?kiosk=true` or just `?kiosk`.

this mode will hide both navigation bars and disable the command palette (press escape to leave this mode as before) 👍

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This question has been answered so I will be closing this discussion. Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
